pyproject.toml

```diff
@@ -34,7 +34,7 @@ examples = [
     "scikit-image",
     "trimesh",
     "gltflib",
-    "imgui-bundle>=1.2.1",
+    "imgui-bundle>=1.6.0",
```

Comment:
this can be reverted right? i feel like we loosened the requirements on wgpu-py

Reply:
i guess its for the examples only, and users won't get pip complaining so its probably a no-op.